WeakAuras/WeakAuras.lua

```diff
@@ -78,6 +78,48 @@ function WeakAuras.InternalVersion()
   return internalVersion;
 end
 
+do
+  local currentErrorHandlerId
+  local currentErrorHandlerUid
+  local currentErrorHandlerContext
+  local function waErrorHandler(errorMessage)
+    local prefix = ""
+    local data
+    if currentErrorHandlerId then
+      data = WeakAuras.GetData(currentErrorHandlerId)
+    elseif currentErrorHandlerUid then
+      data = Private.GetDataByUID(currentErrorHandlerUid)
+    end
+    if data then
+      Private.AuraWarnings.UpdateWarning(data.uid, "LuaError", "error",
+        L["This aura has caused a Lua error."] .. "\n" .. L["Install the addons Bugsack and BugGraber for detailed error logs."])
```

Comment:
```suggestion
      Private.AuraWarnings.UpdateWarning(data.uid, "LuaError", "error",
        L["This aura has caused a Lua error."] .. "\n" .. L["Install the addons BugSack and BugGrabber for detailed error logs."])
```

Reply:
`/console scriptErrors 1` is not *that* bad to recommend, i don't know state of blizz'error frame for all versions, if i remember right on DF alpha it's paged like bugsack

Reply:
What about turning the 5th arg of UpdateWarning `printOnConsole` on? It's made to not spam

Reply:
Done that. It'll probably lead to lots of complaining, similar to the warnings about protected clones. But whatever.

Reply:
I did?